lib/lang/isEmpty.js

```diff
@@ -7,23 +7,23 @@ var hasValues = require('has-values');
  * functions, numbers, strings, nulls, objects and arrays.
  *
  * ```js
- * utils.lang.isEmpty('a');
- * //=> true
- *
- * utils.lang.isEmpty('');
+ * isEmpty('a');
  * //=> false
  *
- * utils.lang.isEmpty(1);
+ * isEmpty('');
```

Comment:
actually, yeah that gave me an idea. this is great how it is then. thanks for making them consistent.